<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>584</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Rzeczy w miejscach 0.3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="zakladki">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="aktywne">
     <attribute name="title">
      <string>Aktywne</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>160</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wszystkie">
     <attribute name="title">
      <string>Wszystkie</string>
     </attribute>
     <widget class="QTableWidget" name="wszystkieTablica">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>741</width>
        <height>481</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>Nowy wiersz</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>id=0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>id=2</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Rzeczy</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Miejsca</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Czasy</string>
       </property>
      </column>
      <item row="1" column="0">
       <property name="text">
        <string>Chuj</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>Tutaj</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="flags">
        <set>ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>Złoto</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>Tam</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="flags">
        <set>ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="wszystkieUsun">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>500</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuń</string>
      </property>
     </widget>
     <widget class="QPushButton" name="wszystkieEdytuj">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>500</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Edytuj</string>
      </property>
     </widget>
     <widget class="QPushButton" name="wszystkieDodaj">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>500</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
     <widget class="QPushButton" name="wszystkieAktywuj">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Aktywuj</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="kreator">
     <attribute name="title">
      <string>Kreator</string>
     </attribute>
     <widget class="QGroupBox" name="creatorEditorGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>761</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>Edytor</string>
      </property>
      <widget class="QPushButton" name="creatorEditorDeleteFromMacro">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>480</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Usuń z makro</string>
       </property>
      </widget>
      <widget class="QTreeWidget" name="creatorEditorActions">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>211</width>
         <height>411</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Czynności</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Akcje myszki</string>
        </property>
        <item>
         <property name="text">
          <string>Przemieść kursor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Kliknięcie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Podwójne kliknięcie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Przytrzymaj przycisk myszy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Puść przycisk myszy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ruch kółka myszy</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Akcje klawiatury</string>
        </property>
        <item>
         <property name="text">
          <string>Kliknij klawisz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Użyj skrótu klawiszowego</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Przytrzymaj klawisz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Puść klawisz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wypisz tekst</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Puść wszystkie klawisze</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Logika</string>
        </property>
        <item>
         <property name="text">
          <string>Czekaj na akcję myszy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Czekaj na akcję klawiatury</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Czekaj N sekund</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wykonaj N razy</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Nagrane</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Makra</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="creatorEditorAddToMacro">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>450</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Dodaj do makro</string>
       </property>
      </widget>
      <widget class="QPushButton" name="creatorEditorMoveUp">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>450</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>▲</string>
       </property>
      </widget>
      <widget class="QPushButton" name="creatorEditorMoveDown">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>480</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>▼</string>
       </property>
      </widget>
      <widget class="QPushButton" name="creatorEditorSave">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>480</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Zapisz makro</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="creatorEditorName">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>450</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>450</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nazwa makra</string>
       </property>
      </widget>
      <widget class="QPushButton" name="testSaveButton">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Zapisz</string>
       </property>
      </widget>
      <widget class="QPushButton" name="creatorEditorClear">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>480</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Wyczyść edytor</string>
       </property>
      </widget>
      <widget class="QPushButton" name="creatorEditorDeleteFromActions">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>480</y>
         <width>103</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Usuń nagranie</string>
       </property>
      </widget>
      <widget class="QPushButton" name="creatorEditorNewRecording">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>450</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Nowe nagranie</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>152</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Lista komend tworzących makro</string>
       </property>
      </widget>
      <widget class="QPushButton" name="creatorEditorEdit">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>480</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Edytuj</string>
       </property>
      </widget>
      <widget class="QPushButton" name="testLoadButton">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>300</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Wczytaj</string>
       </property>
      </widget>
      <widget class="QPushButton" name="testButton">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>340</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QTreeView" name="creatorEditorTreeView">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>471</width>
         <height>411</height>
        </rect>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>200</number>
       </attribute>
      </widget>
      <widget class="QPushButton" name="testButton_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>370</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QPushButton" name="WhatIsThisButton">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>400</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Co to jest?</string>
       </property>
      </widget>
      <zorder>creatorEditorDeleteFromMacro</zorder>
      <zorder>creatorEditorActions</zorder>
      <zorder>creatorEditorAddToMacro</zorder>
      <zorder>creatorEditorMoveUp</zorder>
      <zorder>creatorEditorMoveDown</zorder>
      <zorder>creatorEditorSave</zorder>
      <zorder>creatorEditorName</zorder>
      <zorder>label_9</zorder>
      <zorder>creatorEditorClear</zorder>
      <zorder>creatorEditorDeleteFromActions</zorder>
      <zorder>creatorEditorNewRecording</zorder>
      <zorder>label_8</zorder>
      <zorder>creatorEditorEdit</zorder>
      <zorder>creatorEditorTreeView</zorder>
      <zorder>testSaveButton</zorder>
      <zorder>testLoadButton</zorder>
      <zorder>testButton</zorder>
      <zorder>testButton_2</zorder>
      <zorder>WhatIsThisButton</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="AC_Group">
     <attribute name="title">
      <string>Autoclicker</string>
     </attribute>
     <widget class="QGroupBox" name="AC_Time">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>321</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Odstęp czasowy między kliknięciami</string>
      </property>
      <widget class="QSpinBox" name="AC_Hours">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="AC_Minutes">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="AC_Seconds">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="AC_Miliseconds">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>54</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Milisekundy</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>53</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Sekundy</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>51</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Minuty</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>53</y>
         <width>38</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Godziny</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="AC_Start">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AC_Stop">
      <property name="geometry">
       <rect>
        <x>194</x>
        <y>270</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="AC_ClickHowManyTimesGroup">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>161</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Ile razy ma klikać</string>
      </property>
      <widget class="QRadioButton" name="AC_ClickNTimes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Klikaj</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="AC_ClickNTimesN">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>razy</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="AC_ClickUntilStopped">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>123</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Klikaj do zatrzymania</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Opcje</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>113</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Wybierz przycisk myszy</string>
       </property>
      </widget>
      <widget class="QComboBox" name="AC_WhichButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Lewy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Środkowy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Prawy</string>
        </property>
       </item>
      </widget>
      <widget class="QKeySequenceEdit" name="AC_Hotkey">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="keySequence">
        <string>Ctrl+Q</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Skrót klawiszowy</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="settings">
     <attribute name="title">
      <string>Ustawienia</string>
     </attribute>
     <widget class="QCheckBox" name="settingsAutosave">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>157</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Autozapis po każdej zmianie</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsDefault">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Przywróć domyślne ustawienia</string>
      </property>
     </widget>
     <widget class="QPushButton" name="forceSave">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Zapisz</string>
      </property>
     </widget>
     <widget class="QPushButton" name="forceLoad">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>440</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Wczytaj</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
